.travis.yml

```diff
@@ -1,14 +1,12 @@
 language: rust
 rust:
-  - nightly-2019-09-03
```

Comment:
so i don't recommend remove this, but update it to a newer working version is fine